app/controllers/UpscanController.scala

```diff
@@ -106,6 +106,10 @@ class UpscanController @Inject()(authAction: AuthAction,
       } else {
         Future.successful(Redirect(routes.CheckingServiceController.checkCSVFilePage()))
       }
+    } recover {
+      case e: Exception =>
+        logger.error("[UpscanController][successCSV] Failed to update the ids: " + e.getMessage)
```

Comment:
instead of concat the exception can you do it in Interpolation

Reply:
updated